CorsixTH/Lua/game_ui.lua

```diff
@@ -750,6 +752,29 @@ function GameUI:onMouseWheel(x, y)
   return UI.onMouseWheel(self, x, y)
 end
 
+--! Announcements to the player.
+--!param msgs (array of string) Messages to select from.
+--!param rnd_frac (optional float in range (0, 1]) Fraction of times that the
+--    call actually says something.
+--!param priority (optional string - Low, Normal, High, Critical) Priority of announcement
+--!return (boolean) Whether a message was given to the user.
+function GameUI:makeAnnouncement(msgs, rnd_frac, priority)
+  if not priority or not AnnouncementPriority[priority] then priority = "Normal" end
```

Comment:
Not clear why `playAnnouncement` takes a priority and this takes a key into AnnouncementPriority. It's not used that way anyway, so let's assume it's already an AnnouncementPriority that gets passed in.

Reply:
I was planning to use it that way once its possible, eg [epidemic start/end](https://github.com/CorsixTH/CorsixTH/blob/master/CorsixTH/Lua/epidemic.lua#L228). I'm using the key to avoid needing to include the AnnouncementPriority at the top of each file where its used (or where you wanted to set any later parameters).